src/programQueueProcessor/unit.test.ts

```diff
@@ -254,8 +254,8 @@ describe("kafka integration", () => {
 
   describe("programQueueProcessor", () => {
     it.only("must index all clinical and RDPC data into Elasticsearch", async () => {
-      createIndexAndAlias(TEST_US);
-      createIndexAndAlias(TEST_CA);
+      await createIndexAndAlias(TEST_US);
+      await createIndexAndAlias(TEST_CA);
```

Comment:
if these are two independent operations, can do `await Promise.all([createIndexAndAlias(TEST_US), createIndexAndAlias(TEST_CA)])` to parallelize